### PR TITLE
Convert existing policies to operator policies for installing operators

### PR DESCRIPTION
**Pull request by zyjjay**
Converted policies:
- Compliance operator policy
- Quay container security operator policy
- Gatekeeper operator policy

ref: https://issues.redhat.com/browse/ACM-10573

### CODE REVIEW

File: frontend/src/wizards/Governance/Policy/PolicyWizard.tsx [hunk 1]
```diff
@@ -439,7 +439,7 @@ export function PolicyWizardTemplates(props: { policies: IResource[] }) {
                 label={t('Namespace')}
                 labelHelp={t('The operator is installed in this namespace.')}
               />
-              <WizTextInput path="objectDefinition.spec.subscription.channel" label={t('Channel')} required />
+              <WizTextInput path="objectDefinition.spec.subscription.channel" label={t('Channel')} />
```
Comment (mprahl) [inline frontend/src/wizards/Governance/Policy/PolicyWizard.tsx:442]: This shouldn't be necessary once you rebase on the latest main branch due to this PR merging:
https://github.com/stolostron/console/pull/3422

**Review by mprahl — COMMENTED** [review]
This is so much simpler! I'll approve this once you rebase the PR.

File: frontend/src/wizards/Governance/Policy/stable/CA-Security-Assessment-and-Authorization/policy-compliance-operator-install.yaml [hunk 2]
```diff
@@ -17,79 +17,25 @@ spec:
...
-          remediationAction: inform # will be overridden by remediationAction in parent policy
+          remediationAction: inform
           severity: high
-          object-templates:
-            - complianceType: musthave
-              objectDefinition:
-                apiVersion: v1
-                kind: Namespace
-                metadata:
-                  name: openshift-compliance
```
Comment (JustinKuli) [inline frontend/src/wizards/Governance/Policy/stable/CA-Security-Assessment-and-Authorization/policy-compliance-operator-install.yaml:33]: Shouldn't the policy still create this namespace?

Comment (zyjjay) [reply frontend/src/wizards/Governance/Policy/stable/CA-Security-Assessment-and-Authorization/policy-compliance-operator-install.yaml:33]: I was under the impression that the current version of the controller will create the ns if it doesn't already exist in enforce mode. Is this not the case?

Comment (mprahl) [reply frontend/src/wizards/Governance/Policy/stable/CA-Security-Assessment-and-Authorization/policy-compliance-operator-install.yaml:33]: It doesn't.

**Review by JustinKuli — APPROVED** [review]
/unhold

**Pull request merged**